governance-app/src/signUpdateValidatorScoreParameters.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _CONCORDIUM_APP_UPDATE_VALIDATOR_SCORE_PARAMETERS_H_
+#define _CONCORDIUM_APP_UPDATE_VALIDATOR_SCORE_PARAMETERS_H_
+
+/**
+ * Handles the signing flow, including updating the display, for the 'update validater score parameters'
+ * update instruction.
+ * @param cdata please see /doc/ins_update_validator_score_parameters.md for details
+ */
+void handleSignUpdateValidatorScoreParameters(uint8_t *cdata, volatile unsigned int *flags);
+
+typedef struct {
+    uint8_t max_missed_rounds[21];
```

Comment:
Why 21 bytes here? I expected 8 for the `uint64`.
```suggestion
    uint8_t max_missed_rounds[8];
```

Reply:
The value put into the array is not `uint64` though, but rather a text representation of a `uint64`, so 21 `char`s. I did not invent this structure btw; just following what other updates do 😄